src/include/module/module/base.h

```diff
@@ -30,6 +30,10 @@ struct module_config {
 	const void *init_data; /**< Initial IPC configuration. */
 #if CONFIG_IPC_MAJOR_4
 	struct ipc4_base_module_cfg base_cfg;
+	uint8_t nb_input_pins;
+	uint8_t nb_output_pins;
```

Comment:
nitpick: is "nb" an abbreviation for "number?" If so, it seems a bit non-standard to me, maybe just "n" or "num" or even "number_of..."

Reply:
It's the pre-existing name for the field in struct ipc4_base_module_cfg_ext from which this is derived.  Personally I don't like it much either, but I'm trying to be a good citizen and not rock the boat too much.  :)